src/overlays/actors/ovl_En_Fu/z_en_fu.c

```diff
@@ -73,14 +72,26 @@ static Vec3f D_80964B0C = { 0.0f, 60.0f, -8.0f };
 static Vec3f D_80964B18 = { 0.0f, 55.0f, 12.0f };
 static Vec3f D_80964B24 = { 0.0f, 60.0f, 0.0f };
 
-static AnimationInfo sAnimationInfo[] = {
-    { &gHoneyAndDarlingIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
-    { &gHoneyAndDarlingCupCheeksLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
-    { &gHoneyAndDarlingHugLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
-    { &gHoneyAndDarlingGameDanceLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },
+typedef enum {
+    /* 0 */ HONEY_DARLING_ANIM_IDLE,
+    /* 1 */ HONEY_DARLING_ANIM_CUP_CHEEKS,
+    /* 2 */ HONEY_DARLING_ANIM_HUG,
+    /* 3 */ HONEY_DARLING_ANIM_GAME_DANCE,
+    /* 4 */ HONEY_DARLING_ANIM_HOLD_HANDS_MORPH,
+    /* 5 */ HONEY_DARLING_ANIM_HOLD_HANDS,
+    /* 6 */ HONEY_DARLING_ANIM_SURPRISE,
+    /* 7 */ HONEY_DARLING_ANIM_MAX
+} HoneyAndDarlingAnimation;
+
+static AnimationInfo sAnimationInfo[HONEY_DARLING_ANIM_MAX] = {
+    { &gHoneyAndDarlingIdleAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },          // HONEY_DARLING_ANIM_IDLE
+    { &gHoneyAndDarlingCupCheeksLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f }, // HONEY_DARLING_ANIM_CUP_CHEEKS
+    { &gHoneyAndDarlingHugLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // HONEY_DARLING_ANIM_HUG
+    { &gHoneyAndDarlingGameDanceLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, 0.0f },  // HONEY_DARLING_ANIM_GAME_DANCE
+    // HONEY_DARLING_ANIM_HOLD_HANDS_MORPH
     { &gHoneyAndDarlingHoldHandsLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },
```

Comment:
The enum comment on a different line is a bit annoying. What about using this comment style for all the entries of this array instead?

Reply:
Sure. Some of these animations once named can get quite long, so I could see this being a recurring issue, just something to think about